Readme.md

```diff
@@ -72,6 +72,11 @@ Of first run it will create "GroupName-Algorithm.txt" files for each group that
 Mixed rigs (AMD+NVIDIA) are not recommended. Use at your own risk.
 You may have problems with some miners selecting specific devices. Disable problematic miners if you have issues.
 
+#### Nvidia-SMI
+
+nvidia-smi.exe is included as an optional tool to modify gpu/memory clocks and other details of your Nvidia GPU(s), it can be initiated in the "miningparameters.json" 
```

Comment:
```it can be initiated in the "miningparameters.json"```
this in incorrect.
there is no relation between miningparameters.json and nvidia-smi

Reply:
ahhhhhhh. there is a user asking about using that...... I was under the impression thats how you initialized it. does it have to be run in start.bat to be run?

Reply:
just yanked out that sentence. or we could yank the whole thing. no matter